allure-generator/src/main/javascript/components/side-nav/allure_logo.svg

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
-<svg viewBox="0 0 500 500" xmlns="http://www.w3.org/2000/svg">
+<svg viewBox="0 0 500 500" xmlns="http://www.w3.org/2000/svg" preserveAspectRatio="xMinYMid">
   <path d="M 4.077 343.174 L 158.19 497.287 L 226.24 429.237 L 72.139 275.138 L 4.25 343.026 L 4.238 343.014 Z M 292.137 494.587 L 495.795 290.931 L 429.15 224.285 L 360.725 292.71 L 206.222 138.206 L 274.647 69.782 L 207.632 2.768 L 4.235 206.165 L 72.459 274.391 L 148.336 198.515 L 302.84 353.018 L 226.705 429.155 Z" style="fill: rgb(255, 37, 37);" transform="matrix(-1, 0, 0, -1, 499.871994, 500.054988)"/>
 </svg>
```

Comment:
Here is a cool website:
https://jakearchibald.github.io/svgomg/

Paste your svg and you will get smaller version with of the same content